packages/ember-application/lib/system/application.js

```diff
@@ -316,46 +295,19 @@ var Application = Namespace.extend(DeferredMixin, {
     Create a container for the current application's registry.
 
     @private
-    @method buildContainer
+    @method buildInstance
     @return {Ember.Container} the configured container
   */
-  buildContainer: function() {
+  buildInstance: function() {
     var container = this.__container__ = this.__registry__.container();
```

Comment:
Should we maintain `__container__` on the `Application` as well as the `ApplicationInstance`? Removing it from the `Application` might not be worth it until 2.0, since it is so frequently accessed despite being "private" (e.g. from the Ember Inspector). Perhaps we should flag it for 2.0 removal?